packages/docusaurus/src/webpack/plugins/CleanWebpackPlugin.ts

```diff
@@ -30,6 +30,7 @@
 // More context: https://github.com/facebook/docusaurus/pull/1839
 
 import path from 'path';
+import * as fs from 'fs';
```

Comment:
Use `fs-extra` and `await fs.pathExists()`; you can find other examples.

Reply:
@Josh-Cena I'm pushing a commit that use await pathExists instead of sync func